src/assets/v1.3.0/compatibility_fallback_handler.json

```diff
@@ -17,6 +17,7 @@
     "1285": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "1287": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "4002": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
+    "4690": "0xc6Dd460C2a3f4cf74C2D213B4CB81c6943DA7793",
```

Comment:
Which commit from the contracts repo did you use for deployment? The expected address should be either   `0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4` or `0x017062a1dE2FE6b99BE3d9d37841FeD19F573804`

Reply:
Hello @rmeissner I used [https://github.com/gnosis/safe-contracts/tree/1.3.0-libs.0](https://github.com/gnosis/safe-contracts/tree/1.3.0-libs.0), but as @mikheevm mentioned here:
[https://discord.com/channels/477106835862716416/914975477389721611/914979975105495100](https://discord.com/channels/477106835862716416/914975477389721611/914979975105495100)

I had to set the `deterministicDeployment` property to false

![image](https://user-images.githubusercontent.com/19674406/144238433-15c591ff-5b43-46eb-8f1e-7950a2d0ab43.png)

on all the deployment scripts:
![image](https://user-images.githubusercontent.com/19674406/144238622-cf2b063f-6224-47de-bdd0-dfb9106e23f3.png)

because the chain doesn't accept deterministic factory. Please @mikheevm correct me if I'm wrong about that.